LWCOSC/force-app/main/default/lwc/onlineShopIndexPage/onlineShopIndexPage.js

```diff
@@ -0,0 +1,346 @@
+import { LightningElement,track } from 'lwc';
+
+import updatePurchaseOrderRecords from '@salesforce/apex/ProductController.updatePurchaseOrderRecords';
+import Id from '@salesforce/user/Id';
+
+
+export default class OnlineShopIndexPage extends LightningElement {
+    purchaseOrderFlag=true;
+	ispurchaseOrderBtnDisabled = false;
+    productListFlag;
+    isNewPurchaseBtnDisable;
+    cartDataFlag;
+    priorData=[];
+    cartData=[];
+	cartErrors = [];
+	invoiceDataFlag;
+	@track invoiceCartData = [];
+	invoiceDate;
+	invoice;
+	userId = Id;
+
+    historyHandler(event){
```

Comment:
the name of the handlers should be "handleXXXX", as a handler is just a method and the method should be named as a verb rather than a noun or an adjective.

Reply:
Point Noted, As discussion No action require